### DISCUSSION TITLE
Codespaces Down?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24617] [Apps, API and Webhooks]
<p>Getting the unicorn page when trying to list codespaces right now.  Nothing acknowledged on the GitHub Status page.</p>
<p>What’s up?</p>
<p><a href="https://github.com/codespaces" rel="noopener nofollow ugc">Unicorn! · GitHub</a><br>
<strong>This page is taking too long to load.</strong></p>

**Comment by ghost** [comment]
<p>Came back up a while later.</p>
<p>I’m new to codespaces, is this something that happens often or did I just get ‘lucky?’</p>

**Comment by mpboom** [comment, marked as answer]
<p>GitHub has a public status page:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/19948032/181085668-a4b61063-1d45-47da-b9f6-fb2651fe33bc.png" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/" target="_blank" rel="noopener nofollow ugc">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://www.githubstatus.com/" target="_blank" rel="noopener nofollow ugc">GitHub Status</a></h3>

  <p>Welcome to GitHub's home for real-time and historical data on system performance.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<aside class="quote no-group" data-username="jason-hobbs_shawinc" data-post="2" data-topic="251649">
<div class="title">
<div class="quote-controls"></div>
 jason-hobbs_shawinc:</div>
<blockquote>
<p>is this something that happens often or did I just get ‘lucky?’</p>
</blockquote>
</aside>
<p>You can judge that for yourself, the incident history is public :wink:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/history">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/19948032/181085670-84f08276-b1fa-45de-b062-2865c5f1f534.png" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/history" target="_blank" rel="noopener nofollow ugc">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://www.githubstatus.com/history" target="_blank" rel="noopener nofollow ugc">GitHub Status - Incident History</a></h3>

  <p>GitHub's Incident and Scheduled Maintenance History</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
